docs/examples/example_satellite_observations.ipynb

```diff
@@ -0,0 +1,360 @@
+{
```

Comment:
<p>Above this cell, we could write out the equations for MKE and EKE (in markdown / latex).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocean-eddy-cpt/gcm-filters/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>